firmware/application/apps/ui_recon.cpp

```diff
@@ -830,6 +860,8 @@ ReconView::ReconView(NavigationView& nav)
 
     field_wait.on_change = [this](int32_t v) {
         wait = v;
+        if (wait == -100)
```

Comment:
What is this doing? Maybe add a comment to explain why?

Reply:
Added // replacing -100 by 200 in SPEC_MODULATION because else it's freezing the device